manifest_template.yml

```diff
@@ -78,7 +78,7 @@ apigee:
 {% set portal_visibility = ENV.portal_visibility | default(true) %}
 {% set TITLE = 'e-Referral Service' %}
 {% set NAME = 'e-referrals-service-api-' + ENV.name %}
-{% if ENV.has_variants == 'true' %}
+{% if ENV.name == 'internal-dev' or ENV.name == 'dev' %}
```

Comment:
Are we no longer using has_variants then? i.e. defined above.

Reply:
I have a vague recollection it was a review comment to use env name directly. But maybe need to clean up config.